notebook/static/notebook/js/codecell.js

```diff
@@ -509,10 +509,10 @@ define([
     };
 
 
-    CodeCell.prototype.clear_output = function (wait) {
-        this.output_area.clear_output(wait);
-        this.set_input_prompt();
+    CodeCell.prototype.clear_output = function (wait, ignore_queue) {
```

Comment:
It's fine the way it is. `undefined` is treated as `false`, which means things should proceed as normal in the `clear_output` function.